scripts/pylib/twister/twisterlib/handlers.py

```diff
@@ -39,7 +39,7 @@
 logger = logging.getLogger('twister')
 logger.setLevel(logging.DEBUG)
 
-SUPPORTED_SIMS = ["mdb-nsim", "nsim", "renode", "qemu", "tsim", "armfvp", "xt-sim", "native"]
+SUPPORTED_SIMS = ["mdb-nsim", "nsim", "renode", "qemu", "tsim", "armfvp", "xt-sim", "native", "misc"]
```

Comment:
is misc appropriate here? Why not `custom`?

Reply:
I followed the convention of misc flasher, I think `custom` would work just as well